android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -474,6 +458,7 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
     const val QUESTIONNAIRE_CONFIG = "questionnaire-config"
     const val QUESTIONNAIRE_COMPUTED_VALUES_MAP = "computed-values-map"
     const val QUESTIONNAIRE_ACTION_PARAMETERS = "action-parameters"
+    const val STRING_INTERPOLATION_PREFIX = "@{"
```

Comment:
Don't we have this Constant in a universal constants file? I think it's used in several places.

Reply:
I'm not sure whether it is. I cannot seem to trace it. In the S[tring.Interpolate method](https://github.com/opensrp/fhircore/blob/70489e14827789efe8096196e452b8a206a03996/android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/StringExtensions.kt#L42) its specified explicitly